scripts/fixup_headers.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env python3
+
+# This script checks and can optionally update FENIX source files.
+# You should always run this script without the "-u" option
+# first to make sure there is a clean dry run of the files that should
+# be updated
+# This is based on a script of the same name in the MOOSE Framework:
+# https://github.com/idaholab/moose/blob/master/framework/scripts/fixup_headers.py
+
+import os, string, re, shutil
+from optparse import OptionParser
+
+global_dir_ignores = ['contrib', '.svn', '.git', 'crane', 'moose', 'squirrel']
```

Comment:
```suggestion
global_dir_ignores = ['contrib', '.svn', '.git', 'moose']
```

Removing remnants from the Zapdos origin of this file.

Reply:
Good catch!